### PR TITLE
Update News Style for News Widget

### PR DESCRIPTION
**Pull request by KimberleyChong**
Updates the style of the news entries to:

- display the date the article was published
- add dividing lines
- limit to 4 articles instead of 5
- match the layout to the [mocks](https://www.figma.com/file/MyPLx5xoj2FCqO47jR23ZD/New-Dashboard%3A-News-and-Twitter-Widget-Designs?node-id=0%3A1&t=IAC09Y8o2Z1sZ69V-1)

`news.html` will be embedded in the new `news-box.component` to display on the new dashboard:

image

Embedded on the homepage without the box:
image

Ticket: [SEAB-5200](https://ucsc-cgl.atlassian.net/browse/SEAB-5200?atlOrigin=eyJpIjoiODVlNjU4NjQyZmI0NDdlYmEwYTVhY2NjZWE3YmY2MTEiLCJwIjoiaiJ9)

[SEAB-5200]: https://ucsc-cgl.atlassian.net/browse/SEAB-5200?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by denis-yuen — APPROVED** [review]

**Review by coverbeck — COMMENTED** [review]
Sort of torn on this. The original idea was to keep this as plain html, making it easy for anybody to edit, and have the styling done by the ui2. When we had non-engineering curators contributing to this repo, they could just edit the content in plain HTML and not have to worry about styling. Now they'll have to, although I'll admit it's not that hard to copy and paste the blocks.

What do you think about doing this programmatically in the UI? Fetch the plain HTML, get each `<span>` element, insert the `<mat-divider>`s and have a stylesheet for the rest?

The other extremely minor disadvantage is that when this gets merged, it gets deployed. I think develop is only used by staging/qa, and master is used by prod, but there will be a tiny, tiny window where this is not in sync with the UI

1. We merge this first to master, the old UI is pulling this in
2. We merge this after deploying the new UI, the new UI is pulling in the old, unstyled content.

But we should be able to keep that window fairly small, so it's not a huge deal.

**Comment by denis-yuen** [comment]
> When we had non-engineering curators contributing to this repo, they could just edit the content in plain HTML and not have to worry about styling. Now they'll have to, although I'll admit it's not that hard to copy and paste the blocks.

FWIW, I was thinking this. I think neo-Ash or neo-Beth should be ok copy-pasting the blocks with the `class` tags intact. 
Especially if they end up programming in WDL, CWL like last time, this seems like a pretty short hurdle.

File: news.html
Comment (coverbeck) [inline news.html:49]: Indentation is off, line should start on column 0 :)

**Review by coverbeck — APPROVED** [review]
We should take a look at staging once this deployed to see what it looks like.

**Review by kathy-t — APPROVED** [review]
Is there a ticket for this? If so, can you add a link to it in the description. Thanks!

File: news.html
Comment (svonworl) [inline news.html:4]: Not necessary to change, because I think it works in [almost?] all the browsers, but fwiw, this is illegal HTML, see https://ucsc-cgl.atlassian.net/browse/DOCK-2116

**Review by svonworl — APPROVED** [review]

File: news.html
Comment (KimberleyChong) [reply news.html:4]: TIL ^ Going to change them to `div`s now since I don't see this file being updated in the near future...

**Pull request merged**
